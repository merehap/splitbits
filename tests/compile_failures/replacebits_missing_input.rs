use splitbits::*;

fn main() {
    let a = 0b101u16;
    let b = 0b00001u8;
    let c = 0b0101u128;
    let d = false;
    replacebits!(overflow=truncate, "aaab bbbb .d.. cccc")
}
