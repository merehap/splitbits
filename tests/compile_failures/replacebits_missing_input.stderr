error: proc macro panicked
 --> tests/compile_failures/replacebits_missing_input.rs:8:5
  |
8 |     replacebits!(overflow=truncate, "aaab bbbb .d.. cccc")
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Either an input or template was missing, but found a setting instead.
